src/client/pydaos/__init__.py

```diff
@@ -37,6 +34,7 @@ def __init__(self, message, rc):
     def __str__(self):
         return self.message
 
+
```

Comment:
I notice a lot of additional new lines. Is this a change in the python coding style?